content/docs/packit-service.md

```diff
@@ -90,6 +90,14 @@ for the particular check:
 
 ![Re-run Github check](/github-check-rerun.png)
 
+Or it is possible to re-trigger every failed task using pull request command
+
+    /packit rebuild-failed
+
+which builds only against failed builds and similar for testing farm
+
+    /packit retest-failed
+
```

Comment:
```suggestion
which builds only failed builds and similar for testing farm

    /packit retest-failed

to <explicitly say what it does>
```

Reply:
you left the `against`, which I'd suggested to remove. Did you overlook it or am I missing something?

Reply:
True. Thanks, Jirko!